combat-trainer.lic

```diff
@@ -2306,15 +2306,15 @@ class AttackProcess
       weapon_name = game_state.wield_offhand_weapon(skill)
 
       actions = if skill.include?('Thrown')
-                  %w[lob lob lob]
+                  %w['lob left' 'lob left' 'lob left']
```

Comment:
💡 The `%w` syntax doesn't work for multiple-word-phrases. I suspect you want `['lob left', 'lob left', 'lob left']`